wicket-core/src/main/java/org/apache/wicket/markup/head/ResourceAggregator.java

```diff
@@ -353,6 +353,7 @@ private void renderCombinedEventScripts()
 		}
 		if (combinedScript.length() > 0)
 		{
+			combinedScript.append("\nWicket.Event.publish(Wicket.Event.Topic.AJAX_TIMERS_BOUND);");
```

Comment:
It follows the same logic as the [AJAX_HANDLERS_BOUND](https://github.com/apache/wicket/blob/master/wicket-core/src/main/java/org/apache/wicket/markup/head/ResourceAggregator.java#L339) event.

Which is always fired if there's `OnDomReadyHeaderItem`s.

If the event name is confusing, then I'd recommend changing both of them to suit their actual function.